roles/edpm_nova/templates/run-on-host

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec nsenter -a -t 1 -- `realpath -s $0` "$@"
```

Comment:
This new file should be copied to edpm host in update.yml[1], otherwise minor update would break. I'm fixing it in https://github.com/openstack-k8s-operators/edpm-ansible/pull/1054/commits/55891ad956fff6071d327f2ffc0f77ee401b60dd

[1] https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_nova/tasks/update.yml#L2